charts/nri-bundle/README.md.gotmpl

```diff
@@ -7,6 +7,27 @@
 
 {{ template "chart.homepageLine" . }}
 
+## Bundled charts
+
+This chart do not deploy anything by itself but has many charts as dependencies install and upgrade the New Relic Kubernetes
+Integration using only one chart.
+
+In case you need more information of each component this chart installs or you are and advanced user that want to install each
+component separated, here is a list of components that this chart installs and where you can have more information about them:
+
+| Component                    | Installed by default? | Github repository                                                                                                                                      |
```

Comment:
I think this table is very useful! Do you think it would also make sense to have a short sentence explaining which things are monitored by each chart?